pom.xml

```diff
@@ -131,12 +131,14 @@
         <plugin>
           <groupId>org.apache.felix</groupId>
           <artifactId>maven-bundle-plugin</artifactId>
+          <version>5.1.1</version>
```

Comment:
Version should come from parent pom and it should be `5.1.1` for bundle plug-in already, so this is redundant I think?